sass/components/ClusterOverview/health.scss

```diff
@@ -1,16 +1,30 @@
-.kubevirt-health__body {
-  text-align: center;
-  max-height: 130px;
+/* stylelint-disable plugin/selector-bem-pattern */
+.kubevirt-cluster-overview__health-grid {
```

Comment:
can we update `StorageOverview/ocs-health.scss` for consitency? or unite these two together into one file?

Reply:
@matthewcarleton I've updated snapshots etc to make the build pass. As @suomiy said, can you please update `ocs-health.scss` in this PR too ? For other changes I'd suggest to create a new branch and new PR to divide these changes to smaller pieces.

Reply:
Yes, I can do that. Thanks @suomiy

Reply:
Sorry this got larger than it should have. I'll create a new PR for the following updates.